lib/plausible_web/plugs/logger_metadata_plug.ex

```diff
@@ -0,0 +1,62 @@
+defmodule PlausibleWeb.LoggerMetadataPlug do
+  require Logger
+
+  @behaviour Plug
+
+  def init(opts), do: opts
+
+  def call(conn, _opts) do
+    start = System.monotonic_time()
```

Comment:
Any special reason for using `System.monotonic_time` here?

Reply:
It looks like it measures time most accurately (https://hexdocs.pm/elixir/1.12/System.html#module-time)